app/src/main/java/com/example/molliestephenson/tictactoe/MainActivity.java

```diff
@@ -2,12 +2,62 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.widget.Button;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.GameEngine;
+import ttt.game.Marks;
 
 public class MainActivity extends AppCompatActivity {
 
+    private List<Button> mBoard = new ArrayList<>();
+    private GameEngine mGameEngine;
```

Comment:
@felipesere @sl4m : From what I have seen from the Udacity course and Android book, the m prefix is Android convention.  From the android code style docs: "Non-public, non-static field names start with m" (https://source.android.com/source/code-style.html#follow-field-naming-conventions) I know that the Udacity course and Android book are a bit old, but since it is still in the docs I kept it.